src/bootstrap/compile.rs

```diff
@@ -1390,6 +1390,18 @@ impl Step for Sysroot {
         let _ = fs::remove_dir_all(&sysroot);
         t!(fs::create_dir_all(&sysroot));
 
+        // When bootstrap relies on a more recent version of LLVM than the beta compiler,
```

Comment:
"bootstrap relies on a more recent version of LLVM" -- can you say more about what you mean by relies on here? How are we relying on LLVM from bootstrap? The linked issue seems like it's not about bootstrap relying but rather that stage0 compiler tests do, if I intuited right at a glance?

(Code seems OK, but a comment update would be good).